atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -1305,4 +1325,20 @@ default Runnable close() {
         TransactionService transactionService();
         Optional<TransactionSchemaInstaller> schemaInstaller();
     }
+
+    @Value.Immutable
+    @Value.Style(stagedBuilder = false)
+    public interface RemoteProxies<T> {
```

Comment:
🎉

Reply:
I actually didn't write this at first, I just had the two Ts, and then when doing a self-review I thought this would be nice